bokeh/plotting_helpers.py

```diff
@@ -125,6 +125,7 @@ def _process_sequence_literals(glyphclass, kwargs, source):
         kwargs[var] = var
 
 def _make_glyph(glyphclass, kws, extra):
+        if extra is None: return None
```

Comment:
What's the need for this? I fell I'm missing context here. In case extra is `None` I'd expect you still want to call `glyphclass(**kws)` and just skip `kws.update(extra)`